virtual_ecosystem/models/animal/animal_cohorts.py

```diff
@@ -133,7 +127,41 @@ def metabolize(self, temperature: float, dt: timedelta64) -> float:
         # in data object
         return actual_mass_metabolized * self.individuals
 
-    def excrete(
+    def excrete(self, excreta_mass: float, excrement_pool: DecayPool) -> None:
+        """Transfers nitrogenous metabolic wastes to the excrement pool.
+
+        This method will not be fully implemented until the stoichiometric rework. All
+        current metabolic wastes are carbonaceous and so all this does is provide a link
+        joining metabolism to a soil pool for later use.
```

Comment:
Should this have a `TODO` so it is easier to find and amend when relevant?

Reply:
I've going to do a pass through all of my code to rework the TODOs and make issues from them shortly, I'll add this to the list.